mux/bindings/python/cmux_mux_client/client.py

```
@@ -0,0 +1,541 @@
+        self.close()
+
+    def close(self) -> None:
+        self._conn.close()
+
+    def _next_id(self) -> int:
+        with self._id_lock:
+            value = self._next_request_id
+            self._next_request_id += 1
+            return value
+
+    def request(self, cmd: str, **params: Any) -> Dict[str, Any]:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**params`

(ANN401)